src/components/atoms/Icon/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import * as svg from './svg'
+
+export const iconTypes: IconType[] = Object.keys(svg) as any[]
```

Comment:
여기 `any[]` 타입 어썰션은 꼭 필요한 부분인가요??? 👀

Reply:
요거 어썰션 안 걸면 `string[]` 으로 나와서 `IconType[]`에 타입 바인딩이 안 되겠군여...ㅜㅜ 어썰션이 필요하긴 하겠네용
그럼 요런 방법은 어떠실까요??? 🙏

```suggestion
export const iconTypes = Object.keys(svg) as IconType[];
```

Reply:
https://github.com/Lubycon/clelab-frontend/pull/7/commits/aa5229c9ec56f2ca9568c0009e6c6d40661abec3 에서 수정하였습니다.